### PR TITLE
add note about 21Vianet

### PR DESCRIPTION
**Pull request by VLG17**
https://github.com/MicrosoftDocs/microsoft-365-docs/issues/1406

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - please note; the asterisk is displayed as a bullet point in rich text view.

**Comment by VLG17** [comment]
> Copy review - please note; the asterisk is displayed as a bullet point in rich text view.

@JohanFreelancer9 how can i make it show as asterisk then? Could you help me with that please?

File: microsoft-365/admin/dns/create-dns-records-at-cloudflare.md [hunk 1]
```diff
@@ -161,8 +161,9 @@ When Office 365 finds the correct TXT record, your domain is verified.
     |CNAME  <br/> |lyncdiscover  <br/> |webdir.online.lync.com  <br/> |30 minutes  <br/> |
     |CNAME  <br/> |enterpriseregistration  <br/> |enterpriseregistration.windows.net  <br/> |30 minutes  <br/> |
     |CNAME  <br/> |enterpriseenrollment  <br/> |enterpriseenrollment-s.manage.microsoft.com  <br/> |30 minutes  <br/> |
-    |CNAME  <br/> |msoid  <br/> |clientconfig.microsoftonline-p.net  <br/> |30 minutes  <br/> |
-    
+    |CNAME  <br/> |msoid*  <br/> |clientconfig.microsoftonline-p.net  <br/> |30 minutes  <br/> |
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/dns/create-dns-records-at-cloudflare.md]:
```suggestion
    |CNAME  <br/> |msoid<sup>**1**</sup>  <br/> |clientconfig.microsoftonline-p.net  <br/> |30 minutes  <br/> |
```

File: microsoft-365/admin/dns/create-dns-records-at-cloudflare.md [hunk 2]
```diff
@@ -161,8 +161,9 @@ When Office 365 finds the correct TXT record, your domain is verified.
...
+
+* Applies to Office 365 operated by 21Vianet.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/dns/create-dns-records-at-cloudflare.md]:
```suggestion
<sup>**1**</sup> Applies to Office 365 operated by 21Vianet.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@VLG17  My suggestion.

**Comment by mypil** [comment]
@VLG17 - please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

File: microsoft-365/admin/dns/create-dns-records-at-cloudflare.md [hunk 3]
```diff
@@ -161,8 +161,9 @@ When Office 365 finds the correct TXT record, your domain is verified.
     |CNAME  <br/> |lyncdiscover  <br/> |webdir.online.lync.com  <br/> |30 minutes  <br/> |
     |CNAME  <br/> |enterpriseregistration  <br/> |enterpriseregistration.windows.net  <br/> |30 minutes  <br/> |
     |CNAME  <br/> |enterpriseenrollment  <br/> |enterpriseenrollment-s.manage.microsoft.com  <br/> |30 minutes  <br/> |
-    |CNAME  <br/> |msoid  <br/> |clientconfig.microsoftonline-p.net  <br/> |30 minutes  <br/> |
-    
+    |CNAME  <br/> |msoid<sup>**1**</sup>  <br/> |clientconfig.microsoftonline-p.net  <br/> |30 minutes  <br/> |
+
+<sup>**1**</sup> Applies to Office 365 operated by 21Vianet.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/dns/create-dns-records-at-cloudflare.md]:
```suggestion
    <sup>**1**</sup> Applies to Office 365 operated by 21Vianet.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Apologies @VLG17 . Please add spaces.

**Comment by mypil** [comment]
@pebaum - PR has been copy edited and is ready for final review. This is based on issue #1406. 

Thank you.


cc: @kenwith

**Comment by pebaum** [comment]
We've decided not to include this guidance for clarity in the global walkthrough articles based on customer and engineering feedback.

**Pull request closed without merging**